frontend/src/pages/onboarding/index.tsx

```diff
@@ -119,7 +131,7 @@ export default function Onboarding() {
   const { handleSubmit, trigger, watch } = methods;
 
```

Comment:
Would be good to have "Next" button become "Skip" when all fields in the form are optional AND there is no input (or empty input) for all the fields (For instance, on the Grades/Certification pages)